query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/commons/LdapMessages.java

```diff
@@ -70,6 +72,10 @@ public static ErrorMessage userNameAttributeNotFoundError(List<String> path) {
         return new ErrorMessageImpl(USER_NAME_ATTRIBUTE_NOT_FOUND, path);
     }
 
+    public static ErrorMessage mappingAttributeNotFoundError(List<String> path) {
```

Comment:
Could we refactor the "USER_NAME_ATTRIBUTE_NOT_FOUND" to "USER_ATTRIBUTE_NOT_FOUND" and reuse it here instead of creating a new code?